package.json

```diff
@@ -166,7 +165,6 @@
     "classnames": "^2.2.5",
     "color-hash": "^2.0.1",
     "comlink": "^4.3.0",
-    "cross-fetch": "^3.1.5",
```

Comment:
This lib was introduced to added a polyfill for `airbrake-js`. We have switched to `@airbrake/browser` in the meantime, which seems to work without polyfills.